quic/transport/packets/packetnumber.nim

```diff
@@ -2,10 +2,10 @@ import std/math
 import pkg/stew/endians2
 
 type
-  PacketNumber* = range[0'u64..2'u64^62-1]
+  PacketNumber* = range[0'i64..2'i64^62-1]
```

Comment:
I must admit that this `2^62-1` confuses me. It must be something Quic related but it's really unusual. Is there a doc for this? Is this normal? Why not use 2^63-1?

Reply:
I didn't write this code, this PR is just making it work with i386. I believe the spec is https://datatracker.ietf.org/doc/html/rfc9000#name-variable-length-integer-enc. I haven't reviewed if this implementation is correct.